src/admin-views/tribe-options-upgrade.php

```diff
@@ -0,0 +1,121 @@
+<?php ob_start(); ?>
```

Comment:
Consider moving the `ob_start();` and `ob_get_clean();` to the `Tribe__Events__Main::do_upgrade_tab()` method so the template need only be concerned with template logic and the method is concerned with how to output it.

Reply:
We should switch into using `Tribe__Template` class to load this.